modules/ossm-routing.adoc

```diff
@@ -119,11 +127,11 @@ Destination rules are applied after virtual service routing rules are evaluated,
 [id="ossm-routing-lb_{context}"]
 ==== Load balancing options
 
-By default, {ProductName} uses a round-robin load balancing policy, where each service instance in the instance pool gets a request in turn. {ProductName} also supports the following models, which you can specify in destination rules for requests to a particular service or service subset.
+By default, {ProductName} uses a round-robin load balancing policy, where each service instance in the pool gets a request in turn. {ProductName} also supports the following models, which you can specify in destination rules for requests to a particular service or service subset.
 
 * Random: Requests are forwarded at random to instances in the pool.
 * Weighted: Requests are forwarded to instances in the pool according to a specific percentage.
-* Least requests: Requests are forwarded to instances with the least number of requests.
+* Least: Requests are forwarded to instances with the least number of requests.
```

Comment:
According to https://istio.io/latest/docs/concepts/traffic-management/, this should be "Least requests"